bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/ItemUtil.java

```diff
@@ -75,14 +75,18 @@ public static void assertValidItemName(String itemName) throws IllegalArgumentEx
         }
     }
 
-    public static State convertToAcceptedState(final State state, final Item item) {
+    public static State convertToAcceptedState(State state, Item item) {
+        if (state == null) {
```

Comment:
I wonder how `null` actually gets in here. IMHO we should fix the root cause of it first. Still, it's valid to make it fail-safe here.